src/tests/functional/plugin/gpu/shared_tests_instances/set_device_name.cpp

```diff
@@ -5,23 +5,26 @@
 #include <algorithm>
 #include <stdexcept>
 
-#include <functional_test_utils/ov_plugin_cache.hpp>
+// #include <functional_test_utils/ov_plugin_cache.hpp>
 
 #include "set_device_name.hpp"
 
 namespace ov {
 namespace test {
 
 void set_device_suffix(const std::string& suffix) {
-    std::string new_gpu_name = CommonTestUtils::DEVICE_GPU;
-    new_gpu_name += CommonTestUtils::DEVICE_SUFFIX_SEPARATOR;
-    new_gpu_name += suffix;
-    auto available_devices = utils::PluginCache::get().core()->get_available_devices();
-    if (std::find(available_devices.begin(), available_devices.end(), new_gpu_name) == available_devices.end()) {
-        throw std::runtime_error("The device " + new_gpu_name + " is not in the available devices! Please use other on!");
-    }
+    static std::string new_gpu_name;
+    new_gpu_name =
+        std::string(CommonTestUtils::DEVICE_GPU) +
+        std::string(1, CommonTestUtils::DEVICE_SUFFIX_SEPARATOR) +
+        suffix;
+    // TODO: fix me (device suffix validation)
```

Comment:
Please uncomment! It works (have checked with @vladimir-paramuzov )

Reply:
Fixed.